src/Modal/Modal.scss

```diff
@@ -109,3 +109,123 @@
 .modal-footer {
   flex: 0 0 auto;
 }
+
+.pgn__modal-popup__tooltip {
```

Comment:
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/2828721/174297022-73c6bdc8-ccb2-4be2-aef4-6a70b7f20201.png">

Might be another z-index issue going on with how the `ModalPopup` is rendered, and the z-index on the header within the Paragon documentation site. I think I'd expect the header to be the top most layer.

Reply:
Corrected